tool/planet/etcd.go

```diff
@@ -153,13 +153,24 @@ func etcdBackup(backupFile string) error {
 
 // etcdDisable disables etcd on this machine
 // Used during upgrades
-func etcdDisable(upgradeService bool) error {
+func etcdDisable(upgradeService, stopAPIServer bool) error {
 	ctx, cancel := context.WithTimeout(context.Background(), EtcdUpgradeTimeout)
 	defer cancel()
 
+	// Kevin: Workaround, for the API server presenting stale data to clients while etcd is down. Make sure we shut down
+	// the API server as well (passed as flag from gravity to prevent accidental usage).
+	// TODO: This fix needs to be revisited to include a permanent solution.
+	if stopAPIServer {
+		err := systemctl(ctx, "stop", APIServerServiceName)
```

Comment:
Is this going to return an error if the apiserver is already stopped (e.g. on non-active leader) node, or it won't?

Reply:
Doesn't seem to:
```
kevin-test4:/$ systemctl stop kube-apiserver
kevin-test4:/$ echo $?
0
kevin-test4:/$ systemctl stop kube-apiserver
kevin-test4:/$ echo $?
0
```

Reply:
I don't think so:
```
kevin-test4:/$ systemctl stop kube-apiserver
kevin-test4:/$ echo $?
0
kevin-test4:/$ systemctl stop kube-apiserver
kevin-test4:/$ echo $?
0
```